template/packages/backend/test/app.e2e-spec.ts

```diff
@@ -1,24 +1,96 @@
-import { Test, TestingModule } from '@nestjs/testing';
-import { INestApplication } from '@nestjs/common';
-import request from 'supertest';
-import { AppModule } from './../src/app.module';
+import { INestApplication } from "@nestjs/common"
+import { Test, TestingModule } from "@nestjs/testing"
+import {
+  Config,
+  startPostgresContainer,
+} from "@originate/docker-await-postgres"
+import request from "supertest"
+import { createConnection } from "typeorm"
 
-describe('AppController (e2e)', () => {
-  let app: INestApplication;
+jest.setTimeout(15_000)
```

Comment:
Do you think we still need this timeout? I remember we added when were trying to solve a problem. Are you expecting that tests on CI might take a while if they are spinning up a docker container?

Reply:
Yes, I think we still need this. The default timeout is 5 seconds, and that's not enough on my machine. We still haven't tested on OS X.